src/main/java/org/testeditor/fixture/host/HostDriverFixture.java

```diff
@@ -82,11 +82,39 @@ public Status getStatus() {
         return connection.getStatus();
     }
 
+    /**
+     * Provides a possibility to type a value into a specified field through the
+     * parameters row an column.
+     * <p>
+     * Attention! The input field has to be unprotected and not hidden. If they
+     * are, the s3270 emulation will lock further actions.
+     *
+     * @param value
+     *            the value to be typed into the input field of a mainframe
+     *            window.
+     * @param row
+     *            the row of the input field
+     * @param col
+     *            the column of the input field.
+     */
     @FixtureMethod
-    public void typeInto(String value, int row, int col) {
-        logger.info("Set cursor position ...");
-        connection.typeInto(value, new Locator(col, row));
+    public void typeAt(String value, int row, int col) {
+        setCursorPosition(row, col);
+        waiting(100);
+        connection.doCommand("String(\"" + value + "\")");
+        connection.doCommand("ascii"); // just to see if typed in successfully.
+    }
+
+    private void waiting(long time) {
+        try {
+            Thread.sleep(time); // 1000 milliseconds is one second.
```

Comment:
the comment indicates that the time unit is confusing. Instead of the comment the parameter should be called `milliseconds`.